### PR TITLE
Added instruction to install Frigate on QNAP NAS.

### PR DESCRIPTION
**Pull request by my-umd**

### CODE REVIEW

File: docs/docs/frigate/installation.md [hunk 1]
```diff
@@ -277,3 +277,69 @@ You need to configure 2 paths:
 
 ![image](https://user-images.githubusercontent.com/4516296/232585872-44431d15-55e0-4004-b78b-1e512702b911.png)
 
+## QNAP NAS
+
+The instruction was tested on a QNAP with an Intel J3455 CPU and 16G RAM, running QTS 4.5.4.2117.
```
Comment (NickM-27) [inline docs/docs/frigate/installation.md:282]:
```suggestion
These instructions were tested on a QNAP with an Intel J3455 CPU and 16G RAM, running QTS 4.5.4.2117.
```

File: docs/docs/frigate/installation.md [hunk 2]
```diff
@@ -277,3 +277,69 @@ You need to configure 2 paths:
...
+4. Calculate shared memory size according to [documentation](https://docs.frigate.video/frigate/installation).
+5. Find your time zone value from https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+6. ssh to QNAP.
+
+**Installation**
+
+Run the following commands to install Frigate (using v0.12.0 as example):
+```bash
+# Download Frigate image
+docker pull ghcr.io/blakeblackshear/frigate:0.12.0
```
Comment (NickM-27) [inline docs/docs/frigate/installation.md:304]:
```suggestion
docker pull ghcr.io/blakeblackshear/frigate:stable
```

File: docs/docs/frigate/installation.md [hunk 3]
```diff
@@ -277,3 +277,69 @@ You need to configure 2 paths:
...
+  -p 8554:8554 \
+  -p 8555:8555 \
+  -p 8555:8555/udp \
+  --label='com.qnap.qcs.network.mode=nat' \
+  --label='com.qnap.qcs.gpu=False' \
+  --memory="4g" \
+  --cpus="2" \
+  --detach=true \
+  -t \
+  ghcr.io/blakeblackshear/frigate:0.12.0
```
Comment (NickM-27) [inline docs/docs/frigate/installation.md:340]:
```suggestion
docker run \
  --name=frigate \
  --shm-size=256m \
  --restart=unless-stopped \
  --env=TZ=America/New_York \
  --env=LIBVA_DRIVER_NAME=i965 \
  --env=FRIGATE_RTSP_PASSWORD=frigate_password \
  --volume=/share/Container/frigate/config:/config:rw \
  --volume=/share/share_vol2/frigate/media:/media/frigate:rw \
  --network=bridge \
  --privileged \
  --workdir=/opt/frigate \
  -p 1935:1935 \
  -p 5000:5000 \
  -p 8554:8554 \
  -p 8555:8555 \
  -p 8555:8555/udp \
  --label='com.qnap.qcs.network.mode=nat' \
  --label='com.qnap.qcs.gpu=False' \
  --detach=true \
  -t \
  ghcr.io/blakeblackshear/frigate:stable
```

File: docs/docs/frigate/installation.md [hunk 4]
```diff
@@ -277,3 +277,69 @@ You need to configure 2 paths:
...
+mkdir -p /share/share_vol2/frigate/media
+# Create Frigate docker container.  Replace shm-size value with the value from preparation step 3.
+# Also replace the time zone value for 'TZ' in the sample command.
+# Example command will create a docker container that uses at most 2 CPUs and 4G RAM.
+docker run \
+  --name=frigate \
+  --shm-size=256m \
+  --restart=unless-stopped \
+  --env=TZ=America/New_York \
+  --env=LIBVA_DRIVER_NAME=i965 \
```
Comment (NickM-27) [inline docs/docs/frigate/installation.md:322]: this should be explained or removed unless it applies to all potential qnap installations

**Review by NickM-27 — CHANGES_REQUESTED** [review]

File: docs/docs/frigate/installation.md (hunk 4 above)
Comment (my-umd) [reply docs/docs/frigate/installation.md:324]: Removed but added it to the comment.

**Review by NickM-27 — DISMISSED** [review]

**Comment by my-umd** [comment]
Sorry Nick.  I forgot to remove LIBVA_DRIVER_NAME from the sample command before you approved my PR.  I just pushed my change.  May need another touch from you.  I should have no more changes.  Thanks.
    On Saturday, April 22, 2023 at 10:33:44 AM EDT, Nicolas Mowen ***@***.***> wrote:  
 
 


@NickM-27 approved this pull request.

**Review by NickM-27 — APPROVED** [review]

**Pull request merged**